docs/content/reference/03-middleware.md

```diff
@@ -115,38 +115,157 @@ module.exports = {
 
 ### Automatically installed middleware
 
+Automatically-installed middleware is middleware you can configure but do *not* need to attach to
+the app yourself. Boltzmann automatically attaches these middlewares if the features that provide
+them are enabled.
```

Comment:
1. Nit: middleware is `attached`, not `installed`.
2. We might mention that, as opposed to middleware configured at the point of attachment, automatically-attached middleware is configured solely through environment variables.